torchx/lib/torchx/backend.ex

```diff
@@ -1545,26 +1547,16 @@ defmodule Torchx.Backend do
 
   defp window_op(out, tensor, window_dims_tuple, opts, pad_constant, reduce_fun)
        when is_function(reduce_fun, 2) do
-    window_dilations = opts[:window_dilations]
-
-    if window_dilations && window_dilations != List.duplicate(1, tuple_size(tensor.shape)) do
-      raise ArgumentError, "window_dilations unsupported"
-    end
-
-    # if Enum.any?(opts[:padding], fn conf -> conf |> Tuple.to_list() |> Enum.any?(&(&1 != 0)) end) do
-    #   raise ArgumentError, "padding unsupported"
-    # end
-
-    intermediate_type =
-      tensor.type
-      |> Nx.Type.to_floating()
-      |> to_torch_type()
-
     t_tx =
```

Comment:
I'm not sure what was the deal with the intermediate_type casting... The floating point conversion was actually breaking s64 window_min